README.md

```diff
@@ -3,6 +3,46 @@
 Kroxylicious is an exploration for building a Kafka protocol proxy,
 addressing use cases such as multi-tenancy, schema validation, or encryption.
 
+# Setting Up in Windows Using WSL
+### For installing WSL (Skip the First '3' steps if already done)
+1. Enable the Windows Subsystem for Linux feature: To enable WSL, you need to enable the Windows Subsystem for Linux feature in the Windows Features dialog.
+
+
+2. Install a Linux distribution from the Microsoft Store(Ubuntu preferably).
+
+3. Launch the Linux distribution and Configure the Linux environment: After launching the Linux distribution, you can configure the environment by creating a user account and setting the password.
+
+With these steps, you should now have WSL installed and configured on your Windows system. 
+
+For further assistance:https://learn.microsoft.com/en-us/windows/wsl/install
+
+
+4. Open the WSL window and update the packages using
+
+```bash
+  sudo apt update
+  sudo apt upgrade
+```
+5. Install Latest Java version by typing 
+```bash
```

Comment:
This should probably be a [sublist](https://www.markdownguide.org/basic-syntax/#lists-1)